cmd/thor/optimizer/optimizer_test.go

```diff
@@ -146,7 +146,7 @@ func newTempFileDB() (*muxdb.MuxDB, func() error, error) {
 		return nil
 	}
 
-	return db, close, nil
+	return db, closeFun, nil
```

Comment:
```suggestion
	return db, closeFunc, nil
```

Minor, but nice to have :-)